kousa/lib/socket_handler.ex

```diff
@@ -543,8 +544,8 @@ defmodule Kousa.SocketHandler do
   end
 
   def handler("ask_to_speak", _data, state) do
-    with {:ok, room_id} <- Users.tuple_get_current_room_id(state.user_id) do
-      case Kousa.Data.RoomPermission.ask_to_speak(state.user_id, room_id) do
+    with {:ok, room_id} <- Beef.Queries.User.tuple_get_current_room_id(state.user_id) do
+      case RoomsPermission.ask_to_speak(state.user_id, room_id) do
```

Comment:
where is the `RoomsPermission` module defined?  [This pull request](https://github.com/benawad/dogehouse/pull/543) made the identical change which I tried to [address here](https://github.com/benawad/dogehouse/pull/581).